hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieParquetInputFormat.java

```diff
@@ -121,15 +121,15 @@ public RecordReader<NullWritable, ArrayWritable> getRecordReader(final InputSpli
           return super.getRecordReader(split, job, reporter);
         }
         if (supportAvroRead && HoodieColumnProjectionUtils.supportTimestamp(job)) {
-          return new HoodieFileGroupReaderBasedRecordReader((s, j, r) -> {
```

Comment:
reporter was just passed a few levels down and then used without any modification. We can just pass it directly here and simplify the code